lib/src/slices/decision_card_spam/decision_card_spam_service.dart

```diff
@@ -85,13 +92,17 @@ class DecisionCardSpamService extends ChangeNotifier {
     ApiEmailSenderModel? sender =
         await _apiEmailSenderService.getById(senderId);
     if (sender != null) {
+      ApiAuthServiceAccountModel account =
+          await _apiAuthService.getAccountById(sender.accountId!);
       String? mailTo = sender.unsubscribeMailTo;
       if (mailTo != null) {
         String list = sender.name ?? sender.email!;
         await _apiEmailSenderService.markAsUnsubscribed(sender);
         bool unsubscribed = false;
         try {
-          unsubscribed = await _apiGoogleService.unsubscribe(mailTo, list);
+          DataBkgServiceEmail serviceEmail =
+              _dataBkgService.getServiceEmail(account);
+          unsubscribed = await serviceEmail.unsubscribe(mailTo, list);
```

Comment:
this should be setup so it works like this

_dataBkgService.email.unsubscribe(account, mailTo, list);